### PR TITLE
Update iq.m3u

### PR DESCRIPTION
**Pull request by Hani-Poo**

### CODE REVIEW

**Comment by PopeyeTheSai10r** [comment]
Geo-locked or not 24/7

File: streams/iq.m3u [hunk 1]
```diff
@@ -73,6 +73,8 @@ https://cdn3.wowza.com/1/OWNQVXVWc0dQclJz/RXJPdHJx/hls/live/playlist.m3u8
 https://nl2.livekadeh.com/hls2/Bayyinat.m3u8
 #EXTINF:-1 tvg-id="BeitolAbbasTVChannel.iq",BeitolAbbas TV Channel (720p)
 https://live.beitolabbas.tv/live/beitolabbastv.m3u8
+#EXTINF:-1 tvg-id="BeladiSatelliteTV.iq",Beladi Satellite TV (540p)
+http://95.216.180.111:1935/live/68/chunklist_w1016747112.m3u8
```
Comment (Alstruit) [inline streams/iq.m3u:77]:
```suggestion
http://95.216.180.111:1935/live/68/playlist.m3u8
```

**Review by Dum4G — APPROVED** [review]
image

File: streams/iq.m3u [hunk 2]
```diff
@@ -73,6 +73,8 @@ https://cdn3.wowza.com/1/OWNQVXVWc0dQclJz/RXJPdHJx/hls/live/playlist.m3u8
 https://nl2.livekadeh.com/hls2/Bayyinat.m3u8
 #EXTINF:-1 tvg-id="BeitolAbbasTVChannel.iq",BeitolAbbas TV Channel (720p)
 https://live.beitolabbas.tv/live/beitolabbastv.m3u8
+#EXTINF:-1 tvg-id="BeladiSatelliteTV.iq",Beladi Satellite TV (540p)
+http://95.216.180.111:1935/live/68/playlist.m3u8
```
Comment (BellezaEmporium) [inline streams/iq.m3u:77]: Resolved IP 5e085b8c1882e.streamlock.net

https://5e085b8c1882e.streamlock.net/live/68/playlist.m3u8
http://5e085b8c1882e.streamlock.net:1935/live/68/playlist.m3u8

Comment (BellezaEmporium) [reply streams/iq.m3u:77]: If you love digging, there's way more in the link you've found ;)

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**